apps/designer/app/canvas/canvas.tsx

```diff
@@ -65,6 +65,7 @@ const useElementsTree = () => {
     if (rootInstance === undefined) return;
 
     return createElementsTree({
+      embeded: true,
```

Comment:
typo, but also maybe there is a better name, e.g. "sandbox"
```suggestion
      embedded: true,
```

Reply:
Renamed to sandbox

Reply:
if using "embed", correct word would be probably "embed" as a noun